css/properties/font-size-adjust.json

```diff
@@ -5,34 +5,18 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-size-adjust",
           "support": {
-            "chrome": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "43",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "43",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
+            "chrome": {
+              "version_added": "43",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Enable experimental Web Platform features"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": false
+            },
             "edge": [
               {
                 "version_added": "79"
```

Comment:
Let's remove this statement -- Edge 79+ is now based upon Chrome, so it should mirror Chrome data.

Reply:
Can you confirm exactly what bit needs to be removed? (Perhaps using the suggestion feature, then I can just accept it?)

Reply:
The statement that reads just `"version_added": "79"` (without the flag) data should be removed, since Edge 79+ is Chrome-based.  If Chrome doesn't have flagless support, Edge certainly won't!